stacks/Uploaders.ts

```diff
@@ -111,4 +113,36 @@ export function UploadersStack({ stack }: StackContext) {
     });
 
     nptgBucketCdk.addEventNotification(EventType.OBJECT_CREATED, new LambdaDestination(nptgUploader));
+
+    const cleanupRoadworks = new Function(stack, `ref-data-service-cleanup-roadworks`, {
+        bind: [cluster],
+        functionName: `ref-data-service-cleanup-roadworks-${stack.stage}`,
+        handler: "packages/ref-data-uploaders/cleanup-roadworks/index.main",
+        runtime: "nodejs18.x",
+        timeout: 600,
+        memorySize: 3008,
```

Comment:
Can we drop memorySize to 1024?

3008 seems like a lot - think we should start low and bump up if needs be to avoid excessive cost.

Reply:
done